src/index.html

```diff
@@ -4,9 +4,335 @@
+            <a class="nav__link" href="#">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div class="card card--1" data-qa="card">
+          <div class="card__image">
+            <img
+              class="card__imageImac"
+              src="images/imac.jpeg"
+              alt="Моноблок APPLE A1419 iMac"
+            >
+          </div>
+          <div class="card__description">
+            <h1 class="card__product-name">Моноблок APPLE A1419 iMac<br>27" Retina 5K (MNED2UA/A)</h1>
+            <p class="card__info ">Код товара: 195434</p>
+          </div>
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__feedbacks-reviews">Отзывов: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-name">Цена:</p>
+            <p class="card__price-amount">69 999 грн</p>
+          </div>
+          <a
+            class="card__buy-link"
+            href=""
+            data-qa="card-hover"
+          >
+            купить
+          </a>
+        </div>
+
+        <div class="card card--2" data-qa="card">
+          <div class="card__image">
+            <img
+              class="card__imageImac"
+              src="images/imac.jpeg"
+              alt="Моноблок APPLE A1419 iMac"
+            >
+          </div>
+          <div class="card__description">
+            <h1 class="card__product-name">Моноблок APPLE A1419 iMac<br>27" Retina 5K (MNED2UA/A)</h1>
+            <p class="card__info ">Код товара: 195434</p>
+          </div>
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__feedbacks-reviews">Отзывов: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-name">Цена:</p>
+            <p class="card__price-amount">69 999 грн</p>
+          </div>
+          <a
+            class="card__buy-link"
+            href=""
+            data-qa="hover"
+          >
+            купить
+          </a>
+        </div>
+
+        <div class="card card--3" data-qa="card">
```

Comment:
```suggestion
        <div class="card" data-qa="card">
```
Зачем тут card--3?
Используй `nth-child` если надо